src/app/reviewer/[orgSlug]/dashboard/table.tsx

```diff
@@ -0,0 +1,102 @@
+'use client'
+
+import { ActionReturnType } from '@/lib/types'
+import dayjs from 'dayjs'
+import { fetchStudiesForOrgAction } from '@/server/actions/study.actions'
+import { DisplayStudyStatus } from '@/components/study/display-study-status'
+import {
+    Divider,
+    Stack,
+    Title,
+    Table,
+    TableTbody,
+    TableTd,
+    TableTh,
+    TableThead,
+    TableTr,
+    Text,
+    Tooltip,
+    Flex,
+} from '@mantine/core'
+import { FC } from 'react'
+import { Link } from '@/components/links'
+import { useQuery } from '@tanstack/react-query'
+import { StudyJobStatus } from '@/database/types'
+import { Refresher } from '@/components/refresher'
+
+type Studies = ActionReturnType<typeof fetchStudiesForOrgAction>
+
+const Row: FC<{ study: Studies[number]; orgSlug: string }> = ({ study, orgSlug }) => {
+    return (
+        <TableTr key={study.id}>
+            <TableTd>
+                <Tooltip label={study.title}>
+                    <Text lineClamp={2} style={{ cursor: 'pointer' }}>
+                        {study.title}
+                    </Text>
+                </Tooltip>
+            </TableTd>
+            <TableTd>{dayjs(study.createdAt).format('MMM DD, YYYY')}</TableTd>
+            <TableTd>{study.researcherName}</TableTd>
+            <TableTd>{study.reviewerName}</TableTd>
+            <TableTd>
+                <DisplayStudyStatus
+                    studyStatus={study.status}
+                    jobStatus={study.latestJobStatus}
+                    jobId={study.latestStudyJobId}
+                />
+            </TableTd>
+            <TableTd>
+                <Link href={`/reviewer/${orgSlug}/study/${study.id}/review`} c="blue.7">
+                    View
+                </Link>
+            </TableTd>
+        </TableTr>
+    )
+}
+
+const FINAL_STATUS: StudyJobStatus[] = ['CODE-REJECTED', 'JOB-ERRORED', 'RESULTS-APPROVED', 'RESULTS-REJECTED']
+
+export const StudiesTable: FC<{ studies: Studies; orgSlug: string }> = ({ studies: initialStudies, orgSlug }) => {
+    const {
+        data: studies,
+        refetch,
+        isRefetching,
+    } = useQuery({
+        enabled: false,
+        initialData: initialStudies,
+        queryKey: ['org-studies', orgSlug],
+        queryFn: async () => await fetchStudiesForOrgAction({ orgSlug }),
+    })
```

Comment:
Something like...

```suggestion
    const {
        data: studies,
        refetch,
        isRefetching,
    } = useQuery({
        enabled: false,
        initialData: initialStudies,
        queryKey: ['org-studies', orgSlug],
        queryFn: async () => await fetchStudiesForOrgAction({ orgSlug }),
          refetchInterval: (query) => {
            const hasInitiatedStudies = query.state.data.some(s => FINAL_STATUS.includes(s.latestJobStatus || 'INITIATED')
            if (!hasInitiatedStudies) {
              return false
            }

            // keep polling
            return 1000;
      },
    })
```

i wrote this up in the comment editor so syntax may be off, but i wonder if this covers our use case enough 🤔